atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTimestampBackupRunner.java

```diff
@@ -88,7 +88,7 @@ public synchronized long backupExistingTimestamp() {
                             CassandraTimestampUtils.BACKUP_COLUMN_NAME,
                             Pair.create(currentBackupBound, backupValue));
             executeAndVerifyCas(client, casMap);
-            log.info("[BACKUP] Backed up the value {}", currentBackupBound);
+            log.info("[BACKUP] Backed up the value {}", currentBound);
```

Comment:
Is this going to report something funky when the db is empty?  Seems like we might want to explicitly use the backupValue as that is what we are backing up.